source/editor/Widgets/RenderOptions.cpp

```diff
@@ -379,6 +379,22 @@ void RenderOptions::OnTickVisible()
 
                 option_value("Fog density", Renderer_Option::Fog, "Controls atmospheric fog strength", 0.1f);
 
+                if (option("Volumetric Clouds"))
```

Comment:
REDO: renderer options will get moved in their own file and class (RendererOptionsPool.h)